lua/dbee/handler/conn.lua

```diff
@@ -218,10 +218,21 @@ function Conn:layout()
       return {}
     end
 
-    -- sort keys
-    table.sort(layout_go, function(k1, k2)
-      return k1.type .. k1.name < k2.type .. k2.name
-    end)
+    -- sort children of keys in layout
+    for _, lgo in ipairs(layout_go) do
+      if lgo.name == "history" then
+        -- sort descendingly, latest query in the top
+        -- due to table.name is epoch time.
+        table.sort(lgo.children, function(k1, k2)
+          return k1.name > k2.name
+        end)
+      elseif lgo.name == "structure" then
+        -- sort ascendingly, alphabetically
+        table.sort(lgo.children, function(k1, k2)
+          return k1.name < k2.name
+        end)
+      end
+    end
```

Comment:
I wonder if it's really more intuitive to have more recent history near the bottom?

Also, the `.type .. .name` sorts nicer in my opinion, especially if you have views and tables in one "folder" for example

Reply:
`.type .. .name` is definitely worth to group the materialization + name separately; nice feedback!

Regarding the recent history, IMHO, imagine we are growing the amount of results/queries to +1000 and I want to easy go back to my previous execution I'd have to go to the bottom of the history bar to see it, rather than just the top one (which is always at the top) and allow you to easy see it again.

Should probably have some sort of "limit" of the historical view or the ability to delete them if we want - just with the motivation on how the UI tree would look if the amount of queries is growing large 😅

Reply:
[36bea65](https://github.com/kndndrj/nvim-dbee/pull/34/commits/36bea65dcebedcaf9d14c9ec0b3875fb0fea9c22)

The sorting in `History` is still kept the same as the latest result at the top.

The sorting in `Structure` is now done by 
1. alphabetically descending based on the `schemas`
2. Within each schema we sort:
    1. First by the name of each materilization
    2. The name of the materialization (i.e. `VIEW` , `TABLE`, `FUNCTION`, etc...)

Reply:
Nice, didn't think of that. Well, I think my judgement is as good as yours (or worse), so I trust your taste on this one :)